### PR TITLE
wxDVC accessibility tweaks

### PR DESCRIPTION
**Pull request by vslavik**
These improve behavior with JAWS in particular.

The `NotifyEvent` would be much nicer with C++11, of course, but... I'm also a bit worried about it, but it can always be reversed if it causes problems (FWIW, I didn't find any with JAWS or NVDA).

### CODE REVIEW

File: src/msw/ole/access.cpp [hunk 1]
```diff
@@ -1827,8 +1827,28 @@ IAccessible *wxAccessible::GetIAccessibleStd()
 void wxAccessible::NotifyEvent(int eventType, wxWindow* window, wxAccObject objectType,
                         int objectId)
 {
-    ::NotifyWinEvent((DWORD) eventType, (HWND) window->GetHWND(),
-        (LONG) objectType, (LONG) objectId);
+    const HWND hwnd = (HWND)window->GetHWND();
+
+    // send the notification in idle time to be sure it is sent after the change
+    // was fully done in wx code
+    struct SendNotification
```
Comment (vadz) [inline src/msw/ole/access.cpp]: I think this is not going to compile with g++ in non-C++11 mode, template parameters must have extern linkage in C++98.

IOW this struct should be moved out of the function.

**Review by vadz — APPROVED** [review]
No real comments, just as you, I'm not *sure* this is not going to result in any problems, but let's at least experiment with it -- hopefully we'll get more testing of accessibility features now that they actually work for this control.

**Comment by vslavik** [comment]
> hopefully we'll get more testing of accessibility features now that they actually work for this control.

That's debatable, `wxUSE_ACCESSIBILITY` is still off by default and its previous state implies almost nobody enables it. However, now that it's much more lightweight and doesn't normally interfere with Windows' default accessibility implementation, I think it makes sense to enable it by default — should I?

**Comment by vadz** [comment]
Oops, sorry, I was somehow sure that you had already enabled `wxUSE_ACCESSIBILITY` by default now, which was why I thought we'd get more feedback about this code now.

And yes, AFAICS, we really should enable it now as there doesn't seem to be any drawbacks to doing it, are there any I'm missing?

**Comment by vslavik** [comment]
> And yes, AFAICS, we really should enable it now as there doesn't seem to be any drawbacks to doing it, are there any I'm missing?

Not, just a small size increase.

**Pull request merged**